### PR TITLE
Fixed sessions links

### PR DESCRIPTION
**Pull request by AkkarinForest**
I create a new endpoint cause the old one was using as_json. And it was impossible to change it and I need it to return `last_hour_average` now.

### CODE REVIEW

File: app/controllers/measurement_sessions_controller.rb [hunk 1]
```diff
@@ -23,14 +23,19 @@ def show
...
-
-    anchor = "?selectedSessionIds=#{selected_session_ids.to_json}&data=#{data.to_json}"
+    sensorId = "Temperature-airbeam2-f (F)"
 
     if (session.type == "FixedSession")
-      redirect_to fixed_map_path(:anchor => anchor)
+      data =
+        { sensorId: sensorId,
+          isIndoor: session.is_indoor,
+          isStreaming: (session.last_measurement_at > Time.current - 1.hour),
```
Comment (3v0k4) [inline app/controllers/measurement_sessions_controller.rb]: Better to move this to `Session`. Something like `session.is_streaming`

File: app/controllers/measurement_sessions_controller.rb [hunk 2]
```diff
@@ -23,14 +23,19 @@ def show
     session = Session.find_by_url_token(params[:url_token]) or raise NotFound
 
     selected_session_ids = [session.id]
-    data = { sensorId: "Temperature-airbeam2-f (F)" }
-
-    anchor = "?selectedSessionIds=#{selected_session_ids.to_json}&data=#{data.to_json}"
+    sensorId = "Temperature-airbeam2-f (F)"
```
Comment (3v0k4) [inline app/controllers/measurement_sessions_controller.rb]: Is this still a wip?

File: app/controllers/measurement_sessions_controller.rb [hunk 3]
```diff
@@ -23,14 +23,19 @@ def show
...
 
     if (session.type == "FixedSession")
-      redirect_to fixed_map_path(:anchor => anchor)
+      data =
+        { sensorId: sensorId,
```
Comment (3v0k4) [inline app/controllers/measurement_sessions_controller.rb]: Remember to use snake case in Ruby: `sensor_id`

File: app/javascript/angular/code/services/_fixed_sessions.js [hunk 4]
```diff
@@ -82,31 +78,39 @@ export const fixedSessions = (
...
+        this.drawMarkersWithoutLabel(sessionData);
+      }
+    },
     _selectSession: function(id, callback) {
       var sensorId = sensors.selectedId();
       var sensor = sensors.sensors[sensorId] || {};
       var sensorName = sensor.sensor_name;
       if (!sensorName) return;
       $http
-        .get("/api/realtime/sessions/" + id, {
```
Comment (3v0k4) [inline app/javascript/angular/code/services/_fixed_sessions.js]: Can we remove the old endpoint?

File: spec/controllers/api/fixed/sessions_controller_spec.rb [hunk 5]
```diff
@@ -9,7 +9,7 @@
       end_time_local = DateTime.new(2001, 11, 4, 5, 6)
       sensor_name = "sensor-name"
       user = create_user!(username: username)
-      session = create_fixed_session!(user: user, title: title, start_time_local: start_time_local, end_time_local: end_time_local)
+      session = create_fixed_session!({ user: user, title: title, start_time_local: start_time_local, end_time_local: end_time_local })
```
Comment (3v0k4) [inline spec/controllers/api/fixed/sessions_controller_spec.rb]: In ruby you can leave the `{}` out. Internally `fnc(a: 1)` == `fnc({ a: 1 })`

File: app/controllers/measurement_sessions_controller.rb (hunk 3 above)
Comment (AkkarinForest) [reply app/controllers/measurement_sessions_controller.rb]: These are parameters, they have to look like usual params we have in the urls.

File: app/javascript/angular/code/services/_fixed_sessions.js (hunk 4 above)
Comment (AkkarinForest) [reply app/javascript/angular/code/services/_fixed_sessions.js]: Probably, I was planning to look into that in a separate PR.

File: app/controllers/measurement_sessions_controller.rb (hunk 3 above)
Comment (3v0k4) [reply app/controllers/measurement_sessions_controller.rb]: The right side is a ruby var ;)

**Pull request merged**